src/main/resources/org/orbisgis/mapuce/scripts/MApUCE/morphological_indicators.groovy

```diff
@@ -255,36 +259,8 @@ sql.execute "DROP SCHEMA DATA_WORK"
 
 
 
-/**********************/
-/** INPUT Parameters **/
-/**********************/
-
-//Filtrer les tables de type spatial
-
-@DataStoreInput(
-        title = "USR table",
-        resume = "The table to edit",
-        isSpatial = true)
-String usrTable
-
-@DataStoreInput(
-        title = "Roads table",
-        resume = "The table to edit",
-        isSpatial = true)
-String roadsTable
-
-@DataStoreInput(
-        title = "Buildings table",
-        resume = "The table to edit",
-        isSpatial = true)
-String buildingsTable
-
-
 /** String output of the process. */
 @LiteralDataOutput(
         title="Output message",
         resume="The output message")
 String literalOutput
```

Comment:
The variable `literalOutput` is never assign. You should give it a value (even if it is just something like "Work done.") as each WPS process need to have an output.